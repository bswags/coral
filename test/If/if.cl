#cordbconn
server="server";
user="server";
password="server";
port="server";
DBName="server";
type="sqlite";
#enddbconn

#cordb
#enddb

void main() {

	a = 1;

	if (a == 1) {
		a = 2;
		return a;
	}
	if (a < 10 ) {
		a = 12;
		return a;
	}
	if (a > 5 ) {
		a = 4;
		return a;
	}
	printf("derpin");
}
