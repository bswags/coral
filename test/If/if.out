Traceback (most recent call last):
  File "If/if.clx", line 6, in <module>
    setServer("server")
NameError: name 'setServer' is not defined
